src/datajudge/db_access.py

```diff
@@ -632,43 +632,63 @@ def get_row_count(engine, ref, row_limit: int = None):
     return result, [selection]
 
 
-def _column_aggregate(engine, ref, column_operator):
+def _column(
+    engine: sa.engine.Engine,
+    ref: DataReference,
+    aggregate_operator: Callable | None = None,
```

Comment:
```suggestion
    *, aggregate_operator: Callable | None = None,
```
Maybe it'd be good to make this argument keyword-only, which doesn't involve other changes since it's being called always explicitly in all the convenience `get_xx` functions.